x-pack/plugins/apm/server/routes/data_view/create_static_data_view.ts

```diff
@@ -47,11 +58,14 @@ export async function createStaticDataView({
     });
 
     if (!shouldCreateOrUpdate) {
-      return;
+      return {
+        created: false,
+        reason: 'Dataview already exists in current space',
+      };
```

Comment:
For better debugging the endpoint now returns the reason for not creating the data view